src/Tasks/ManifestUtil/mansign2.cs

```diff
@@ -451,6 +460,9 @@ private static void RemoveExistingSignature(XmlDocument manifestDom)
         /// <param name="useSha256">Whether to use sha256</param>
         /// <returns></returns>
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Cryptographic.Standard", "CA5358:RSAProviderNeeds2048bitKey", Justification = "SHA1 is retained for compatibility reasons as an option in VisualStudio signing page and consequently in the trust manager, default is SHA2.")]
+#if RUNTIME_TYPE_NETCORE
+        [SupportedOSPlatform("windows")]
+#endif
```

Comment:
Consider defining `SupportedOSPlatformAttribute` somewhere in our own code for full framework only; then you can drop the ifdef at every use site.